examples/lib/stories/collision_detection/multiple_shapes.dart

```diff
@@ -234,5 +278,27 @@ class MultipleShapes extends BaseGame
       '${fps(120).toStringAsFixed(2)}fps',
       Vector2(0, size.y - 24),
     );
+    const superDebug = false;
+    if (superDebug) {
+      for (final c in components) {
+        if (c is Collidable) {
+          for (final shape in c.shapes) {
+            canvas.renderPoint(shape.absoluteCenter, size: 10);
+            canvas.renderPoint(c.absolutePosition, size: 10);
+            if (shape is Polygon) {
+              final path = Path()
+                ..addPolygon(
+                  (shape as Polygon)
+                      .hitbox()
+                      .map((point) => point.toOffset())
+                      .toList(),
+                  true,
```

Comment:
seems a shape.toPolygon could be useful?

Reply:
Circle can't be defined as a polygon